Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -520,6 +521,9 @@ def rasterize_email_command():
     w, h = check_width_and_height(w, h)  # Check that the width and height meet the safeguard limit
 
     file_name = f'{file_name}.{"pdf" if r_type == RasterizeType.PDF else "jpeg"}'  # type: ignore
+    # force setting the charset in order to ovveride any posible wrong charsets
+    charsets = re.findall(CHARSET_PATTERN, html_body)
+    html_body = html_body.replace(charsets[0], 'UTF-8') if charsets else html_body
```

Comment:
This is dangerous. You may be replacing the charset[0] string in other places in the html_body and not only in the meta tag. You should probably be using `re.sub` with the group option to replace only the matched group.